src/hvm.c

```diff
@@ -1240,7 +1235,7 @@ Port expand(Net* net, Book* book, Port port) {
 
 // Reads back a λ-Encoded constructor from device to host.
 // Encoding: λt ((((t TAG) arg0) arg1) ...)
-Ctr read_ctr(Net* net, Book* book, Port port) {
+Ctr port_to_ctr(Net* net, Book* book, Port port) {
```

Comment:
let's keep the readback functions prefixed by `read_` as a convention

Reply:
What about functions that are inverses of these? I named it `port_to_ctr` b/c I also created a `ctr_to_port`. What do you think we should call the functions that convert `Ctr` to `Port`?

Reply:
I prefer `readback_x` and `inject_x` or `create_x` for the inverses, because it is consistent with how it is in other codes of the past.